server/player/player.go

```diff
@@ -545,9 +547,54 @@ func (p *Player) Hurt(dmg float64, source damage.Source) (float64, bool) {
 		return 0, true
 	}
 
+	w, pos := p.World(), p.Position()
 	totalDamage := p.FinalDamageFrom(dmg, source)
-	damageLeft := totalDamage
+	if ok, _ := p.Blocking(); ok && source.ReducedByArmour() {
+		affected := true
+		if src, ok := source.(damage.SourceEntityAttack); ok {
+			diff := p.Position().Sub(src.Attacker.Position()).Normalize()
```

Comment:
Any reason to normalize this?

Reply:
I don't think so (was taken from Java), removed.